100=ContentProvider.__init__()
101=ContentProvider: %s Loading completed

110=ContentProvider.registerInstance()
111=ContentProvider: registers the instance of a ParameterizedContentProvider with the argument: %s


200=ParameterizedProvider.__init__()
201=ParameterizedProvider%s Loading completed

210=ParameterizedProvider.createContentIdentifier()
211=Identifier: %s -> %s

220=ParameterizedProvider.queryContent()
221=Identitifer: %s ... Completed
222=IllegalIdentifierException: %s
223=Unexpected python exception with traceback: %s

230=ParameterizedProvider.compareContentIds()
231=Identifiers: %s - %s ... seem to be the same
232=Identifiers: %s - %s ... doesn't seem to be the same


300=DataSource.__init__()
301=DataSource: Loading completed

310=DataSource._getUser()
311=Error: Unable to load content with an invalid Identifier: %s
312=Error: Unable to retrieve User from Identifier: %s

320=DataSource._getUserName()
321=Error: Unable to load content without User: %s

330=DataSource.queryClosing()
331=The database: %s has been closed...


400=ContentUser.__init__()
401=Service: %s is not available... Please install the extension: OAuth2OOo!!!
402=Can't retrieve User: %s from provider: network is OffLine!!!
403=User: %s does not exist at this Provider
404=Can't insert User: %s in DataBase
405=ContentUser loading completed

410=ContentUser._getFolderContent()
411=The content of the folder: %s is obtained with Request!!!

420=ContentUser.getDocumentContent()
421=Error: %s - %s


500=Content.__init__()
501=Content loading completed

510=Content._getMetaData()
511=Error: Unable to load content with an incomplete Identifier: %s
512=Error: Unable to retrieve metadata for ID: %s and Uri: %

